packages/tradershub/src/constants/regulators-modal-content.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable sort-keys */
+export type TRowItem = {
+    options?: Record<string, boolean | string>;
+    text: string;
+};
+type TRegulatorContent = Record<string, TRowItem | TRowItem[] | undefined>;
+
+export type TRegulatorsContentProps = {
+    attribute: string;
+    content: TRegulatorContent;
+    id: string;
+    idx?: number;
+};
+
+export const getCFDContents = (): TRegulatorsContentProps[] => [
```

Comment:
Why are we making it into a function? It can be a constant array holder.

Reply:
@aizad-deriv just copied it from appstore package :) but I fully agree with you, I'll change it soon